packages/shared/providers.yaml

```diff
@@ -451,6 +451,13 @@ certn-partner:
             method: GET
             endpoint: /api/v2/teams
     docs: https://docs.nango.dev/integrations/all/certn
+chargebee:
+    categories:
+        - payment
+    auth_mode: BASIC
+    proxy:
+        base_url: https://${connectionConfig.subdomain}.chargebee.com
```

Comment:
With `API_KEY` and `BASIC` auth modes, it is advisable to test the implementation using the `proxy.verification` field.  You can have a look at the following [example](https://github.com/NangoHQ/nango/blob/1381cffd77d4b024011e994d595ef7dc2083222f/packages/shared/providers.yaml#L1675).

Reply:
Could you also mention `api-rate` [limiting](https://apidocs.chargebee.com/docs/api/error-handling?prod_cat_ver=2#api_rate_limits) in the docs and add the `proxy.retry` field to the yaml file. I see the providers docs mentions api rate limiting and it would be good to handle this in [Nango](https://docs.nango.dev/customize/guides/advanced/handle-rate-limits#strategy-2-leverage-rate-limit-headers) 😊. A good example to follow both on docs and the yaml file would be [Klaviyo](https://github.com/NangoHQ/nango/blob/1381cffd77d4b024011e994d595ef7dc2083222f/packages/shared/providers.yaml#L1688)

Reply:
Thanks for letting me know! I've added the required changes please take a look.

Reply:
Strange, I was hoping the `proxy.verification` field would help catch incorrect implementation when creating a new connection. Please omit, 
```
headers:
            Authorization: Basic ${username}
``` 
then try creating a new connection and see if this will be successful. This authentication is already handled in Nango.  Both [bitdefender](https://github.com/NangoHQ/nango/blob/1381cffd77d4b024011e994d595ef7dc2083222f/docs-v2/integrations/all/bitdefender.mdx?plain=1#L30) and chargebee handle authentication the same.

Reply:
Got it works like a charm! Update the `proxy.verification` to `/api/v2/business_entities` this for sure returns empty list if there are no entities. Tested it by creating a new connection and it works.

Reply:
Awesome, glad to hear it works! Regarding the `proxy.verification` field, when creating a new connection: Nango checks if the status is `success` based on the API response status from the verification endpoint. This allows you to determine whether your implementation is correct.